src/rgw/CMakeLists.txt

```diff
@@ -116,6 +116,8 @@ set(librgw_common_srcs
   rgw_role.cc
   rgw_sal.cc
   rgw_sal_filter.cc
+  rgw_redis_driver.cc
+  rgw_ssd_driver.cc
```

Comment:
Should `rgw_redis_driver.cc` be protectected with `if(WITH_RADOSGW_D4N)`? I just tried to build main and got:

/home/ivancich/ceph-work7/ceph/src/rgw/rgw_redis_driver.cc:2:10: fatal error: boost/redis/src.hpp: No such file or directory